person.rb

```diff
@@ -2,14 +2,20 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(id, name = 'Unknown', age = 0, parent_permission: true)
     super()
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
   end
 
```

Comment:
![image](https://github.com/rmiyoyo/oop-school-library/assets/93039359/0be838ea-06f9-4aa9-a4d7-81d36aa1e273)

- [ ] Great work so far but when i try to add the rental in the person object then I face this error which you can see in the image above :eyes: Kindly fix this error to make your association work as expected :+1: